### PR TITLE
fix skipping val metrics

### LINKED ISSUE #10: BUG: validation metrics not logged per epoch, only per batch
When I add the callback and pass validation set or validation split to Keras, validation per-batch metrics are logged under 'test/batch/...', but validation per-epoch metrics are not logged.

These metrics' names begin with `val` and this is filtered out in `impl/__init__.py:23` since merge request #9.

Removing this filtering lets them being logged under `train/epoch/val_...`. But then there is inconsistency between the categories: per-batch is under test, and per-epoch is under train.

It seems that `on_test_batch_end` is invoked after validation, and `on_test_end` is not.

This happens with Tensorflow 2.2.0, and with Tensorflow 2.9.1 as well.

I hope this can be straightened.

### PR DESCRIPTION
**Pull request by Blaizzy**
Closes #10

### CODE REVIEW

**Comment by Blaizzy** [comment]
I changed my mind I'm closing this PR.
